Changelog.md

```diff
@@ -136,7 +136,28 @@
 1. Fixed test - Download world 2
     * [Pull request 110](https://github.com/ignitionrobotics/ign-fuel-tools/pull/110)
 
-### Ignition Fuel Tools 4.X.X (202X-XX-XX)
+### Ignition Fuel Tools 4.5.0 (2022-06-01)
+
+1. APIs for retrieving models in parallel
```

Comment:
I don't see these changes in the forward port changed files, were those included in another tools7 issue? I'm not sure how these changelogs are generated.

Reply:
Oh yeah, #249 and #255 are also being ported in this PR, it's easier to do them all at once.